### PR TITLE
Rename to Firefox Lockbox

### LINKED ISSUE #323: Change instances of "Lockbox" to "Firefox Lockbox"
Based on legal review, change the instances of "Lockbox" to "Firefox Lockbox" throughout the app.

- [x] All copy across any screen or dialog, as updated in the invision file: https://mozilla.invisionapp.com/share/7JEUG47ZU 
- [x] Application name

Additionally, due to line breaks causing issues based on the longer name, we've tweaked content additionally in the following areas:
- [x] **Biometric access onboarding** - Remove the word _"your"_ for both Touch ID and Face ID screens - Handled at #321 / #327 
- [x] **Account > Unlinking description** - Change description to _"This removes synced entries from Firefox Lockbox, but will not delete your account or entries from Firefox."_ - Handled at #305 / #320

### PR DESCRIPTION
**Pull request by devinreams**
Fixes #323 
Two related items already handled at #320 and #327 

Quick pass at finding and replacing standalone instances of "Lockbox" with "Firefox Lockbox". ~Waiting for final inventory of changes from @mozilla-lockbox/ux.~

Note that the name gets a little awkward when trying to distinguish between the overall product vs this iOS app. In docs I called it "Lockbox Firefox for iOS" for now...

### CODE REVIEW

**Comment by nickbrandt** [comment]
@devinreams This global name update as well as a couple copy tweaks based on this change are now captured in #323

**Comment by changecourse** [comment]
Just created #323 to also address some of the changes around this...

**Comment by devinreams** [comment]
The "Firefox Lockbox" app name fits ok on big phones. Less so on small phones. 🤷‍♂️ 

screen shot 2018-05-03 at 1 00 19 pm
iPhone X

screen shot 2018-05-03 at 1 01 21 pm
iPhone SE

**Comment by devinreams** [comment]
@sashei all the renames that happened here may have been too heavy handed. I defer to you on which, if any, should be rolled back.

@mozilla-lockbox/ux this is ready for review and is a sibling to #324. Ideally I can merge that soon and we can review this branch entirely against that _plus_ these string changes and make sure no "Lockbox" instances were overlooked..

File: lockbox-ios/Storyboard/Base.lproj/AccountSetting.storyboard [hunk 1]
```diff
@@ -37,11 +37,11 @@
                                     <constraint firstAttribute="height" constant="44" id="mIY-nI-JMy"/>
                                 </constraints>
                                 <fontDescription key="fontDescription" type="system" pointSize="17"/>
-                                <state key="normal" title="Unlink my Account">
+                                <state key="normal" title="Clear out Firefox Lockbox">
```
Comment (sashei) [inline lockbox-ios/Storyboard/Base.lproj/AccountSetting.storyboard]: You might have made this change in two places...

File: lockbox-ios/Storyboard/Base.lproj/EmptyList.xib [hunk 2]
```diff
@@ -27,7 +27,7 @@
                     <nil key="textColor"/>
                     <nil key="highlightedColor"/>
                 </label>
-                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Look at all this room in your Lockbox!" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="HAm-je-RDU">
+                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Look at all this room in your Firefox Lockbox!" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="HAm-je-RDU">
```
Comment (sashei) [inline lockbox-ios/Storyboard/Base.lproj/EmptyList.xib]: I think this empty state view is going away anyway, but 🤷‍♀️

**Comment by sashei** [comment]
firelock foxbox

**Comment by devinreams** [comment]
🔥 🔒 🦊 📦 

@sashei This rename also seems to result in a name change to the generated schemes in buddybuild (from iOS, Lockbox - Branch to iOS, Firefox Lockbox - Branch). Is that cool?

If so, once this merges, I'll just want all open PRs to get rebase/merged from master and then change the settings in buddybuild to run the new schema instead of the old one.. and then I think we should be in good shape.

**Comment by sashei** [comment]
@devinreams yeah, that's as expected, those names are generated. I'll stay on top of rebasing my branches, maybe we will be able to just click the button in @joeyg 's.

File: lockbox-ios/Storyboard/Base.lproj/AccountSetting.storyboard (hunk 1 above)
Comment (devinreams) [reply lockbox-ios/Storyboard/Base.lproj/AccountSetting.storyboard]: >You might have made this change in two places...

Sure did. Caught it during merge from master / conflict resolution. 👍

**Pull request merged**